examples/async_tx_log.rs

```diff
@@ -25,16 +25,26 @@ async fn main() {
         .await
         .unwrap();
 
-    println!("body = {:?}", body);
-    //  Body = "{:crux.tx/tx-id 7, :crux.tx/tx-time #inst \"2020-07-16T21:50:39.309-00:00\"}"
+    return body;
+}
+
+#[tokio::main]
+async fn main() {
+    let tx_log = tx_log().await;
+    println!("body = {:?}", tx_log);
+    // Body = "{:crux.tx/tx-id 7, :crux.tx/tx-time #inst \"2020-07-16T21:50:39.309-00:00\"}"
+}
+
+#[tokio::test]
+async fn test_tx_log() {
+    let tx_log = tx_log().await;
+    assert!(tx_log.tx___tx_id > 0);
```

Comment:
Again the id > 0, maybe use `is_empty`

Reply:
again any test works here, because it is mostly to detect if there is no inconsistencies with date, not really going to change to is_empty sorry

Reply:
Also `entiity` and `tx_log` don't have a property `is_empty` 🤔

Reply:
is_empty is a String method